TrainingApp/src/data/dummy-data.js

```diff
@@ -0,0 +1,54 @@
+import Product from '../models/product';
```

Comment:
Can we use simple json file instead of js file

Reply:
Helpers are already in place for reducers. So used it here.